.github/workflows/ai-on-demand.yaml

```
@@ -0,0 +1,88 @@
+name: AI On-Demand Assistant
+description: "Provides AI assistance when @efp-dev-ops is mentioned by authorized users"
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)